queries/smali/highlights.scm

```diff
@@ -4,52 +4,106 @@
 
 (primitive_type) @type.builtin
 
-(array_type
-  "[" @punctuation.bracket)
+((class_identifier) @type.builtin
+  (#lua-match? @type.builtin "^Landroid/"))
+((class_identifier) @type.builtin
+  (#lua-match? @type.builtin "^Lcom/android/"))
+((class_identifier) @type.builtin
+  (#lua-match? @type.builtin "^Ldalvik/"))
+((class_identifier) @type.builtin
+  (#lua-match? @type.builtin "^Ljava/"))
```

Comment:
These should be combined.

Reply:
how so? I tried just stacking each lua-match expression in one but then none would work

Reply:
This should work.
```clojure
((class_identifier) @type.builtin
  (#vim-match? @type.builtin "^L(android|com/android|dalvik|java)/"))
```